devstack_vm/bin/collect_logs.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
```

Comment:
Take out the whole list of variables/folders in a config file. Ensure the same config file is used in all locations.